apps/ban-core-writer/src/index.ts

```diff
@@ -92,18 +73,15 @@ const getBanObjectsFromBalRows = (rows: any[]) => {
   const mainToponymes: Record<string, any> = {};
   const districts: Record<string, any> = {};
 
-  const defaultIsoCode = DEFAULT_ISO_CODE; // TODO: replace by district default ISO code if available
-
   rows.forEach((row: any) => {
-    const getterLabels = getLabelsFromRow(row, defaultIsoCode);
     // TODO: Gérer les fusions profondes (plusieurs lignes pour une même entité) cf. oldDistrict
 
     // District
     if (row.id_ban_commune) {
       districts[row.id_ban_commune] = {
         ...districts?.[row.id_ban_commune] || {},
         id: row.id_ban_commune,
-        labels: getterLabels('commune_nom') || [],
+        labels: row.ban_enrich_beautified_labels_commune_nom || [],
```

Comment:
`ban_enrich_beautified_labels_commune_nom` devrait toujours etre un tableau.

```suggestion
        labels: row.ban_enrich_beautified_labels_commune_nom,
```